services/dy-static-file-server/docker/custom/Dockerfile

```diff
@@ -7,31 +7,41 @@ FROM nginx:1.21.0-alpine as base
 #     docker run dy-static-file-server:prod
 #
 
+ARG PYTHON_VERSION="3.8.10-r0"
```

Comment:
I also looked into it, because it looked strage. Apparently you can publish packages to the apk repository like pip. This is the only python version available for this docker image used by Nginx.